app/views/books/index.html.erb

```diff
@@ -21,7 +21,7 @@
           <td> <%= book.title %> </td>
           <td> <%= book.published_date %> </td>
           <td> <%= book.author %> </td>
-          <td> <%= book.price.format %> </td>
+          <%= render "price", book: book %>
```

Comment:
Hound didn't whine about it being double quotes? Nit picking, but I tend to use single quotes or even a symbol in this case leaving double quotes only for interpolation.